server/src/main/java/io/druid/server/initialization/jetty/JettyServerModule.java

```diff
@@ -60,7 +60,7 @@
 import io.druid.server.metrics.DataSourceTaskIdHolder;
 import io.druid.server.metrics.MetricsModule;
 import io.druid.server.metrics.MonitorsConfig;
-import org.apache.http.HttpVersion;
+import io.netty.handler.codec.http.HttpVersion;
```

Comment:
this is a strange change, is it intended?

Reply:
Yes, this module has no longer dependency on HttpClient, and the only usage of `HttpVersion` is to get the version string of HTTP 1.1 (`HttpVersion.HTTP_1_1.toString()`). To reduce confusion, I added a static string variable for HTTP 1.1 version string and removed this import.